src/flash/nor/mspm0.c

```diff
@@ -597,10 +598,6 @@ static int mspm0_fctl_get_sector_reg(struct flash_bank *bank, unsigned int addr,
 		*sector_mask = BIT(sector_num % 32);
 		*reg = FCTL_REG_CMDWEPROTNM;
 		break;
-	case MSPM0_FLASH_BASE_DATA:
-		*sector_mask = BIT((sector_in_bank / 8) % 32);
-		*reg = FCTL_REG_CMDWEPROTB;
```

Comment:
The DATA case will perform the same exact action as MAIN because all devices with DATA bank will use only CMDWEPROTB for protection, which is why the MAIN and DATA are thrown under the same case. This technically is not a formatting change. It's more of condensing.